src/index.html

```diff
@@ -5,9 +5,62 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="../src/styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+
+      <img
+        src="../src/images/imac.jpeg"
+        alt="image"
+        class="card__image"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <h2 class="title">
+        Product code: 195434
+      </h2>
+
+      <div class="review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="price">
+        <p class="price__title">
+          Price:
+        </p>
+
+        <h1 class="cost">
+          $2,199
+        </h1>
```

Comment:
as BEM elements and `p` tag
```suggestion
        <p class="card__price-title">Price:</p>
        <p class="card__price-cost">$2,199</p>
```